content/news/release-20181215-kknd.md

```diff
@@ -0,0 +1,28 @@
+While work continues on the next OpenRA playtest, we have collected a number of bugfixes that we wanted to fix before the holiday season. We are today pushing a bugfix release, [release-20181215](/download/), with the following improvements:
+
+* Fixed servers showing as "Waiting" in the server list after they have started.
+* Fixed a crash in the lobby if the admin changes map while a player is joining.
+* Fixed part of the lobby UI disappearing after transferring admin to another player.
+* Fixed players as "Anonymous" in the lobby after switching from another mod
+* Fixed a crash when using the /levelup cheat with units that are already veteran.
+* Fixed rare crashes when using Attack Move, and when ordering aircraft to land.
+* Fixed bugs with the Chronoshift killing Construction Yards and duplicating MCVs.
+* Fixed with tech-structures if the owner surrenders while it is being captured.
+* Fixed a rare bug where units become uncontrollable after exiting a transport.
+* Fixed compatibilty issues with certain older GPUs.
+* Removed tech-structure abilities from campaign-specific buildings in Red Alert.
+
+Head over to the [downloads page](/download/) to grab the new release, and spread the word!
+
+<hr />
+
+In other news, we are pleased to announce the first public alpha of the [Krush Kill 'n' Destroy remake](https://www.kknd-game.com/) project, which aims to rebuild and improve on the first two KKnD games much like OpenRA has done for the classic C&C games.
```

Comment:
Although most readers probably know or can guess, maybe change the last part before the comma to "(...) project based on OpenRA's engine, (...)" to make clear it's not some unrelated project.